### DISCUSSION TITLE
the problem with users and groups

### DISCUSSION THREAD

**Discussion by DmiRials** [rustdesk/rustdesk-server-pro#172] [General]
I have a problem. I created a user and logged in to one of the clients. 
It is impossible to connect to this client from other clients writes an error
As soon as I log out of my account, I manage to connect to this user.
Users and devices are in the same strategy and group by default.Client versions 1.2.3-1.2.4
What could be the problem?

**Comment by 21pages** [comment]
> It is impossible to connect to this client from other clients writes an error

What's the error?

**Comment by DmiRials** [comment]
> > Невозможно подключиться к этому клиенту от других клиентов пишет ошибка
> 
> В чем ошибка?
[image](https://github.com/rustdesk/rustdesk-server-pro/assets/105746446/0677f6d9-fba1-4d90-b1bc-f3ce96f75a71)

**Comment by 21pages** [comment]
Unbound devices cannot connect to bound devices, need to log in.

**Comment by DmiRials** [comment]
> [#87 (comment)](https://github.com/rustdesk/rustdesk-server-pro/issues/87#issuecomment-1737261023) Unbinded device can't connect to binded device.

I can't connect to the client without a user. Although all the devices are in the same group
Is there any way to connect from a device where user data is entered to a device without a user?

**Comment by 21pages** [comment]
Clients that are not logged in do not belong to any group, clients belong to uses, users belong to groups.
> Is there any way to connect from a device where user data is entered to a device without a user?

A device without a user can be connected to without any setting.
